### PR TITLE
Add PR curves and F1 score to WIT

### PR DESCRIPTION
**Pull request by jameswex**
* Motivation for features / changes

Per user requests, adding PR curves and F1 score to performance tab of WIT for binary classification models.

* Technical description of changes

- Adjust chart generation code to handle both PR curves and existing ROC curves
- Change visuals to handle both charts side by side
- Add calculation of precision (positive predictive value) in order to support PR curve.
- Add calculation of F1 score
- Add row in performance table for F1 score

* Screenshots of UI changes

pr

* Detailed steps to verify changes work correctly (as executed by you)

Ran binary classification demos (single and multimodel) to visualize change and ensure correctness.

### CODE REVIEW

**Comment by jameswex** [comment]
@tolga-b please review

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 1]
```diff
@@ -3779,6 +3883,34 @@ <h2>Show similarity to selected datapoint</h2>
...
+       getF1ModelIndex: function(inferenceStats, thresholds, modelInd, item) {
+        // TODO(jameswex): This unnecessarily recalculates confusion matrix.
+        // Can speed this up.
+        const formatter = d3.format(",.2f");
+        const confCounts = this.getConfusionCountsModelIndex(
+          inferenceStats, thresholds, modelInd, item);
+        if (Object.keys(confCounts).length == 0) {
+          return 0;
+        }
+        if (confCounts['Yes']['Yes'] == 0) {
```
Comment (stephanwlee) [inline tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: might be more readable (or more maintainable) to assign this to a variable.
```js
const truePositive = confCoujnts['Yes']['Yes'];
...
```

Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: done

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 2]
```diff
@@ -1808,6 +1840,11 @@ <h2>Show similarity to selected datapoint</h2>
                                   <div class="perf-table-num-entry">[[getAccuracyModelIndex(inferenceStats_, featureValueThreshold.threshold, index, featureValueThreshold)]]</div>
                                 </template>
                               </div>
+                              <div class="perf-table-f1 perf-table-clickable" on-tap="togglePerfRow">
```
[the remark this answers is not in the document]
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: ack

**Pull request merged**